index.html

```diff
@@ -0,0 +1,208 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CSS Frameworks CA</title>
+    <link rel="stylesheet" href="dist/css/styles.css" />
```

Comment:
```suggestion
    <link rel="stylesheet" href="/dist/css/styles.css" />
```

By using `/dist/` at the start of the link you guarentee that this will always point to `my-project/dist/` whereas using `dist/` this could be different depending on the location of your HTML file.